src/pages/monthly-plan.js

```diff
@@ -675,6 +750,7 @@ export default class MonthlyPlan extends React.Component {
                   className={classnames('button bigButton continueButton', {
                     disableContributionLink,
                   })}
+                  onClick={() => this.saveSelectedDependencies()}
```

Comment:
What's the relationship between `onClick` and `href`? How can you handle `saveSelectedDependencies` if you're not blocking the navigation to the contributionUrl page?

Reply:
`onClick` and `href` I think both work independently, since I started testing I haven't seen or noticed any case where `saveSelectedDependencies` was blocking the navigation to opencollective.

Reply:
In my tests, I'm navigated to the `contributionUrl` and `saveSelectedDependencies` is never triggered. Maybe double check on Firefox if you're not testing with it?

Reply:
Really, this can't work along ... As soon as you navigate, the JavaScript stop being executed and everything that was ongoing is stopped, that's it.

Reply:
I think I'm going to  need help with this, solution I found and tried is to use only `onClick` event and using `window.location.replace` to open the contribution page after saving selected dependencies but I know it is important for us to have `href` and be able to opencollective in a new tab or view the link.

Reply:
1) `preventDefault()` on `onClick`
2) in `onClick`, await for `saveSelectedDependencies` to complete, then redirect using `window.location` to the original `href`